controllers/chat/ChatController.js

```diff
@@ -426,6 +426,135 @@ module.exports = {
       return ErrorResponse.e400(res, error.message);
     }
   },
+  initChatSourceAnonymous: async (req, res) => {
+    const {
+      targetUser,
+      message,
+      anon_user_info_color_code,
+      anon_user_info_color_name,
+      anon_user_info_emoji_code,
+      anon_user_info_emoji_name
+    } = req.body;
+
+    let members = [targetUser];
+    if (!members.includes(req.userId)) members.push(req.userId);
+
+    const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Storing sensitive data like API keys and secrets in environment variables is a good practice, but directly accessing `process.env` throughout your application isn't ideal. It's better to have a central place where you manage your configuration. This way, if you need to change something about the configuration, you only need to do it in one place.

```diff
- const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+ const client = StreamChat.getInstance(config.apiKey, config.secret);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->